sha.go

```diff
@@ -18,6 +18,5 @@ const (
 
 func init() {
 	RegisterAlgorithm(SHA256, crypto.SHA256)
-	RegisterAlgorithm(SHA384, crypto.SHA384)
```

Comment:
This is a breaking change, do we need to bump up the major version?

Reply:
Hm.. good question. Before 084376bb543d4ce80b030a77a6f51f3b3fd861dc (not yet part of a release), neither of them would work out of the box unless the algorithms were registered, so perhaps some wiggle room, but you may be right that it could be a v2 worth if we decide to remove.

Reply:
(honestly not sure why SHA384 and SHA512 ever made it in at all, but 🤷‍♂️)